.github/workflows/add_to_GA.yml

```diff
@@ -0,0 +1,25 @@
+name: Add issues to GA Project
```

Comment:
Let's just add the label `triage` and add the issue to the project.

During triage we should decide whether the issue goes to GA1.0 or not.

Reply:
Okay, looking at the new planned workflow, I will have to make some changes, please ignore this PR- will ping you when I'm ready with the corresponding changes